src/routes/download/+page.svelte

```diff
@@ -23,7 +23,7 @@
 	<p>Patch your favourite apps, right on your device.</p>
 	<div class="buttons">
 		<Query {query} let:data>
-			<Button kind="primary" icon="download" href={data.assets[0].url} target="_blank">
+			<Button kind="primary" icon="download" href={data.assets[0].url} download>
```

Comment:
mimimi the button needs target=_blank or rel=external (will mess up the sveltekit router otherwise).
I don't think download does anything (or does it?).

Reply:
It actually does do something, instead of opening a new tab and closing it, download prop just downloads the file on the same tab.

Reply:
oh ok then